packages/macaque/macaque.0.7.2/descr

```diff
@@ -0,0 +1 @@
+Macaque (Macros for Caml Queries) is a DSL for OCaml, wich produce SQL requests from a comprehension syntax. Macaque can build queries by from simpler components, using phantom types used to ensure safety.
```

Comment:
You may want to add the longer description to the field `Description:` of your `_oasis`.